{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2017-18257",
      "ASSIGNER": "cve@mitre.org"
    },
    "affects": {
      "vendor": {
        "vendor_data": [

        ]
      }
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [

          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "http://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=b86e33075ed1909d8002745b56ecf73b833db143",
          "name": "http://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=b86e33075ed1909d8002745b56ecf73b833db143",
          "refsource": "MISC"
        },
        {
          "url": "https://github.com/torvalds/linux/commit/b86e33075ed1909d8002745b56ecf73b833db143",
          "name": "https://github.com/torvalds/linux/commit/b86e33075ed1909d8002745b56ecf73b833db143",
          "refsource": "MISC"
        },
        {
          "url": "https://www.debian.org/security/2018/dsa-4188",
          "name": "DSA-4188",
          "refsource": "DEBIAN"
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "The __get_data_block function in fs/f2fs/data.c in the Linux kernel before 4.11 allows local users to cause a denial of service (integer overflow and loop) via crafted use of the open and fallocate system calls with an FS_IOC_FIEMAP ioctl."
        }
      ]
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [

    ]
  },
  "impact": {
  },
  "publishedDate": "2018-04-04T17:29Z",
  "lastModifiedDate": "2018-05-03T01:29Z"
}